src/service_aggregator.py

```diff
@@ -530,7 +530,7 @@ async def merge_results_by_node(result_message, merge_qnode):
 
 
 async def robokop_infer(input_message, guid, question_qnode, answer_qnode):
-    automat_url = 'https://automat.renci.org/robokopkg/1.2/query'
+    automat_url = os.environ.get("ROBOKOPKG_URL", "https://automat.renci.org/robokopkg/1.2/")
```

Comment:
Should the default robokopkg url have a `/query` at the end?

Reply:
Ah thanks Max, fixed it